tools/serve/serve.py

```diff
@@ -702,6 +704,63 @@ def start_wss_server(host, port, paths, routes, bind_address, config, **kwargs):
         startup_failed(log=False)
 
 
+class QuicTransportDaemon(object):
+    def __init__(self, host, port, handlers_path=None, private_key=None, certificate=None, log_level=None):
+        args = ["python3", "wpt", "serve-quic-transport"]
+        if host:
+            args += ["--host", host]
+        if port:
+            args += ["--port", str(port)]
+        if private_key:
+            args += ["--private-key", private_key]
+        if certificate:
+            args += ["--certificate", certificate]
+        if handlers_path:
+            args += ["--handlers-path", handlers_path]
+        if log_level == "debug":
+            args += ["--verbose"]
+        self.command = args
+        self.proc = None
+
+    def start(self, block=False):
+        if block:
+            subprocess.call(self.command)
+        else:
+            def handle_signal(*_):
+                if self.proc:
+                    try:
+                        self.proc.terminate()
+                    except OSError:
+                        # It's fine if the child already exits.
+                        pass
+                    self.proc.wait()
+                sys.exit(0)
+
+            signal.signal(signal.SIGTERM, handle_signal)
+            signal.signal(signal.SIGINT, handle_signal)
+
+            self.proc = subprocess.Popen(self.command)
+            # Give the server a second to start and then check.
+            time.sleep(1)
+            if self.proc.poll():
+                sys.exit(1)
+
+
+def start_quic_transport_server(host, port, paths, routes, bind_address, config, **kwargs):
+    # Ensure that when we start this in a new process we have the global lock
+    # in the logging module unlocked
+    reload_module(logging)
+    release_mozlog_lock()
+    try:
+        return QuicTransportDaemon(host,
+                          port,
+                          private_key=config.ssl_config["key_path"],
+                          certificate=config.ssl_config["cert_path"],
```

Comment:
Note that this means when running `wpt serve --quic-transport` we use the certs from `tools/certs` by default, which now works with aioquic 0.8.8.